save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/Card.kt

```diff
@@ -31,21 +34,29 @@ external interface CardProps : PropsWithChildren {
  * @param hasBg adds a white background
  * @param isPaddingBottomNull disables bottom padding (pb-0)
  * @param isNoPadding is true - removes all remaining padding (pt-0 pr-0 pl-0)
+ * @param isFilling
  * @return a functional component representing a card
  */
 @Suppress("EMPTY_BLOCK_STRUCTURE_ERROR")
 fun cardComponent(
     isBordered: Boolean = false,
     hasBg: Boolean = false,
     isPaddingBottomNull: Boolean = false,
-    isNoPadding: Boolean = true
+    isNoPadding: Boolean = true,
+    isFilling: Boolean = false,
 ) = FC<CardProps> { props ->
     val boarder = if (isBordered) "border-secondary" else ""
     val card = if (hasBg) "card" else ""
     val pb = if (isPaddingBottomNull) "pb-0" else ""
     val paddingInside = if (isNoPadding) "pt-0 pr-0 pl-0" else ""
     div {
         className = ClassName("$card card-body mt-0 $paddingInside $pb $boarder")
+        if (isFilling) {
+            style = jso {
+                width = "100%".unsafeCast<Width>()
```

Comment:
isn't this the default value? 100%

Reply:
The width is default, but the height is not. Here I explicitly registered it, so that it is clear.